### PR TITLE
Tof decoding

### PR DESCRIPTION
**Pull request by SzabolcsGergely**

### CODE REVIEW

File: src/device/DeviceBase.cpp [hunk 1]
```diff
@@ -1458,6 +1458,27 @@ std::vector<std::uint8_t> DeviceBase::readFactoryCalibrationRaw() {
...
+    std::vector<uint8_t> eepromDataRaw;
+    std::tie(success, errorMsg, eepromDataRaw) =
+        pimpl->rpcClient->call("readCcmEepromRaw", socket, size, offset).as<std::tuple<bool, std::string, std::vector<uint8_t>>>();
+    if(!success) {
+        throw EepromError(errorMsg);
+    }
+    return eepromDataRaw;
+}
+
+void DeviceBase::writeCcmEepromRaw(CameraBoardSocket socket, std::vector<uint8_t> data, int offset) {
```
Comment (SzabolcsGergely) [inline src/device/DeviceBase.cpp:1473]: should we expose these @themarpe ?

File: include/depthai/pipeline/datatype/ToFConfig.hpp [hunk 2]
```diff
@@ -16,20 +16,13 @@ class ToFConfig : public Buffer {
...
      * Construct ToFConfig message.
      */
     ToFConfig();
     explicit ToFConfig(std::shared_ptr<RawToFConfig> ptr);
     virtual ~ToFConfig() = default;
 
-    ToFConfig& setDepthParams(dai::ToFConfig::DepthParams config);
-    ToFConfig& setFreqModUsed(dai::ToFConfig::DepthParams::TypeFMod fmod);
-    ToFConfig& setAvgPhaseShuffle(bool enable);
-    ToFConfig& setMinAmplitude(float minamp);
```
Comment (moratom) [inline include/depthai/pipeline/datatype/ToFConfig.hpp:32]: These are not needed/useful anymore?

**Review by moratom — APPROVED** [review]

File: include/depthai/pipeline/datatype/ToFConfig.hpp (hunk 2 above)
Comment (SzabolcsGergely) [reply include/depthai/pipeline/datatype/ToFConfig.hpp:32]: No

File: src/device/DeviceBase.cpp (hunk 1 above)
Comment (themarpe) [reply src/device/DeviceBase.cpp:1473]: Lets leave this on `production` for the time being

**Pull request merged**